src/BottomSheet/index.js

```diff
@@ -82,56 +104,90 @@ class BottomSheet extends Component {
+    };
+    return (
+      <>
+        <TouchableOpacity
+          style={styles.background}
+          activeOpacity={1}
+          onPress={() => this.close()}
+        />
+        <Animated.View
+          {...this.panResponder.panHandlers}
+          style={[panStyle, styles.container, { height: animatedHeight }]}
+        >
+          {hasDraggableIcon && (
+            <View style={styles.draggableContainer}>
+              <View
+                style={[
+                  styles.draggableIcon,
+                  {
+                    backgroundColor: dragIconColor || "#A3A3A3"
+                  }
+                ]}
+              />
+            </View>
+          )}
+          {children}
+        </Animated.View>
+      </>
+    )
+  }
+
   render() {
     const {
-      children,
-      hasDraggableIcon,
+      height,
       backgroundColor,
-      dragIconColor
+      useModal = true,
     } = this.props;
-    const { animatedHeight, pan, modalVisible } = this.state;
-    const panStyle = {
-      transform: pan.getTranslateTransform()
-    };
+    const { modalVisible } = this.state;
+
 
     return (
-      <Modal
-        transparent
-        visible={modalVisible}
-        onRequestClose={() => this.setModalVisible(false)}
-      >
-        <View
-          style={[
-            styles.wrapper,
-            { backgroundColor: backgroundColor || "#25252599" }
-          ]}
+      <>
+        {useModal ? <Modal
+          transparent
+          visible={modalVisible}
+          onRequestClose={() => this.setModalVisible(false)}
         >
-          <TouchableOpacity
-            style={styles.background}
-            activeOpacity={1}
-            onPress={() => this.close()}
-          />
-          <Animated.View
-            {...this.panResponder.panHandlers}
-            style={[panStyle, styles.container, { height: animatedHeight }]}
+          <View
+            style={[
+              styles.wrapper,
+              { backgroundColor: backgroundColor || "#25252599" }
+            ]}
           >
-            {hasDraggableIcon && (
-              <View style={styles.draggableContainer}>
-                <View
-                  style={[
-                    styles.draggableIcon,
-                    {
-                      backgroundColor: dragIconColor || "#A3A3A3"
-                    }
-                  ]}
-                />
-              </View>
-            )}
-            {children}
-          </Animated.View>
-        </View>
-      </Modal>
-    );
+            {this._renderAnimatedView({ ...this.props, ...this.state })}
+          </View>
+        </Modal>
+          : modalVisible && this.setModalVisible &&
+          <View
+            style={[
+              styles.wrapper,
+              {
+                backgroundColor: backgroundColor || "#25252599",
+                height: height,
+                backgroundColor: backgroundColor || 'transparent',
+                position: 'absolute',
```

Comment:
Do you think we can change these inline styles for the style file??

Reply:
okay, I will look into it